docs/tractus-x-release.md

```diff
@@ -0,0 +1,21 @@
+# Tractus-X Release
+
+## Artifacts
+
+- Umbrella Helm Chart(s)
+- Changelog (overarching feat)
+
+## Versioning
+
+- Semantic Versioning
+
+## Process
+
+<!-- How are products (considered to be) integrated in the Tractus-X release -->
+- Prior to release, a Tractus-X umbrella Chart release candidate branch is created
+- Producs issue PRs with their release candidate
```

Comment:
This should be the future process, right? currently that's not happening and maybe that should be mentioned.

Reply:
Yes exactly, that's why I opened this draft PR. Jumpstarting this discussion is highly needed in my opinion.
Since this process will rely heavily on committers to support it, we need to come to a basic agreement. 
The two bullet points are just a first idea, that could also make it easy for committers to support that process, since it's a more opt-in focused one.

Right now, I don't really find the time to continue working on the draft